docs/managing-data-quality/recovering-failed-events/manual/testing/index.md

```diff
@@ -6,107 +6,55 @@ sidebar_position: 20
-* `cfg` refers to the configuration you'd like to test;
-* `badrow` refers to an example failed event.
-
-Start by adding your flows, conditions and/or steps as described in the [previous section](/docs/managing-data-quality/recovering-failed-events/manual/configuration/index.md).
-
-Then replace the `badrow` value with a representative failed event example. Be careful with using actual failed events from your production data as it may contain sensitive information. Check with your Data Protection Officer whether you are allowed to copy production data to your local machine.
-
-To run above example, assuming the file is named locally as `test.sc`, run: `amm test.sc`.
-
-This will run the operation that you specified in your configuration (e.g. `replace`) and output the result to the console. You should then be able to check if the payload was modified as expected based on your configuration.
-
-See the [main script](https://github.com/snowplow-incubator/snowplow-event-recovery/blob/master/scripts/Recovery.sc) for additional available convenience functions.
+Which will result in:
+```
+OK! Config valid
+```
 
-## A complete recovery
+Or an detailed error message:
 
-As part of the event recovery project there is the ability to run it in `testOnly` mode. It might be worth while to take a look at the documentation on [running the recovery](/docs/managing-data-quality/recovering-failed-events/manual/running/index.md) first, in order to familiarize yourself with the full process of triggering a recovery job.
+```
+ERROR! Invalid config
 
-You can test a complete recovery, starting from bad rows to getting the data enriched by:
+Instance is not valid against its schema:
+* At $.iglu:com.snowplowanalytics.snowplow.badrows/adapter_failures/jsonschema/1-0-*[0].conditions[0].op:
+  - $.iglu:com.snowplowanalytics.snowplow.badrows/adapter_failures/jsonschema/1-0-*[0].conditions[0].op: does not have a value in the enumeration [Test]
+  ```
 
-- Modifying a `bad_rows.json` file which should contain examples of bad rows you want to recover
-- Adding your recovery scenarios to a `recovery_scenarios.json`
-- If your recovery is relying on specific Iglu repositories additionally to Iglu central, you’ll need to specify those repositories in a `resolver.json`
-- If your recovery is relying on specific enrichments, you’ll need to add them to an `enrichments.json`.
+### Test
 
-Once this is all done, you can run `sbt "project core" "testOnly *IntegrationSpec"`. What this process will do is:
+To run a small-scale recovery test outputting both good and bad events into files, execute:
 
-- Run the recovery on the bad rows contained in `bad_rows.json` according to the configuration in `recovery_scenarios.json`
-- Check that these recovered payloads pass enrichments, optionally leveraging the additional Iglu repositories and enrichments
+```
+snowplow-event-recovery run --config $PWD/scenarios.json -o /tmp -i $PWD/input/ --resolver $PWD/resolver.json
```

Comment:
I'd mention that input is a folder here (rather than a file) and what format the events need to be in here (and any file extensions that are required?).